seranet/Secrets/Encrypt.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Text;
+using System.IO;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Options;
+
+namespace Workplace.Eureka.Services.Secrets
+{
+    public class Encrypt
+    {
+        readonly string PasswordHash ="SampleHash";
```

Comment:
Inject the Secrets to the class via .net DI framework. Refer the code written [Decrypt.cs](https://github.com/99xt/jira-journal/blob/master/seranet/Secrets/Decrypt.cs#L15)